examples/hello-world/package.json

```diff
@@ -43,10 +43,13 @@
   "devDependencies": {
     "@loopback/build": "^1.3.1",
     "@loopback/testlab": "^1.0.7",
-    "@loopback/tslint-config": "^2.0.1",
+    "@loopback/eslint-config": "^1.0.0-1",
     "@types/node": "^10.11.2",
-    "tslint": "^5.12.0",
-    "typescript": "^3.3.1"
+    "eslint": "^5.14.1",
+    "typescript": "^3.3.1",
+    "@typescript-eslint/eslint-plugin": "^1.4.2",
+    "@typescript-eslint/parser": "^1.4.2",
+    "eslint-plugin-eslint-plugin": "^2.0.1"
```

Comment:
Is it necessary to duplicate these three dependencies in every package? 

```
    "@typescript-eslint/eslint-plugin": "^1.4.2",
    "@typescript-eslint/parser": "^1.4.2",
    "eslint-plugin-eslint-plugin": "^2.0.1"
```

Can we find a way how to move them into `@loopback/eslint-config` instead?

Reply:
Unfortunately not yet. See https://github.com/eslint/rfcs/tree/master/designs/2018-simplified-package-loading

Reply:
Yeah, that's rather unfortunate. Thank you for digging the RFC.